code/controllers/subsystem/ai_controllers.dm

```diff
@@ -37,6 +53,15 @@ SUBSYSTEM_DEF(ai_controllers)
 		ai_controller.SelectBehaviors(wait * 0.1)
 		if(!LAZYLEN(ai_controller.current_behaviors)) //Still no plan
 			COOLDOWN_START(ai_controller, failed_planning_cooldown, AI_FAILED_PLANNING_COOLDOWN)
+		if(ai_controller.can_idle)
```

Comment:
Does this mean that people starting a fight with say, megafauna, then run out of their 14 tile reach, means they stop chasing after?

Reply:
if you can get 14 tiles away from a megafauna without it catching up to you I am going to be immensely impressed

Reply:
also all megafauna are still shrimple mobs

Reply:
Not all megafauna are made equal, and even then, they're not basic mobs, so they're still using the simple mob movement timers. Most of them are actually pretty lethargic. Bubblegum at least makes up for it by having a series of rapid teleports to follow after you.

Reply:
we need to think ahead people

Reply:
I've added the ability to configure this on a per-controller basis so the future hypothetical megafauna basic mob AI can utilize it